project-reports/2023/2023-Q2-Hyperledger-Cello.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: 2023 Q2 Hyperledger Cello
+parent: 2023
+grand_parent: Project Updates
+---
+
+# Project Health
+
+V1.0.0 version has been released.
+
+* The new dashboard with operations on users, orgs, channels, and nodes;
+* New agent design and docker agent implementation are done;
+* Finish the API Engine.
+
+# Questions/Issues for the TOC
+
+None
+
+# Releases
+
+* v1.0.0-beta1 is done in November, 2022.
+* v1.0.0-GA has been released in May, 2023.
+
+# Overall Activity in the Past Quarter
+
+The team is working mainly on the remaining work for the 1.0.0-GA release.
```

Comment:
Given that 1.0 is out, it seems like this refers to what happened and should read:
```suggestion
The team was working mainly on the remaining work for the 1.0.0-GA release.
```